### ISSUE TITLE
🐛[BUG] position of outsiders in Box

### ISSUE THREAD

**Issue by mpapenbr** [ant-design/ant-design-charts#800]
### 🐛 bug 
The positioning of outsiders in Box plots seems not correct.

### 📷 
![image](https://user-images.githubusercontent.com/41362745/129172251-2c52b713-842f-44e8-82a8-4a1b9ffd22b6.png)

```
export default function App() {
  const config = {
    data: [
      {
        name: "A",
        minTime: 137.8,
        maxTime: 138.6,
        q1: 138.19,
        median: 138.35,
        q3: 138.42,
        outliers: [137.5, 138.8, 139.2]
      }
    ],
    xField: "name",
    yField: ["minTime", "q1", "median", "q3", "maxTime"],
    outliersField: "outliers",
    yAxis: {
      minLimit: 137,
      maxLimit: 139
    }
  };
  return <Box {...config} />;
}
```
ant-design-charts 1.2.5

Codesandbox: https://codesandbox.io/s/gifted-leavitt-k19lr?file=/src/App.js

**Comment by Shamann** [comment]
I'm not a member of the ant-design team, perhaps they have other opinion. 
Chart components are ment to render the data you provided, so it shouldn't be aware of the actual *_meaning_* of the outliers, they just render the required geometry objects properly positioned on the chart.

**Comment by mpapenbr** [comment]
I agree, I'm responsible in the decision what belongs to the outliers list and what not.
My point is, for example, the outlier 137.5 is not positioned on the 137.5-line, just a bit below (same is true for the other outliers). 
I took the last example from https://charts.ant.design/demos/box as a reference. 
On top: I have a lot of other box plots which are working correct. Some work correct alone, but as soon as a certain other box plot is added to the same graph, it  may happen, that the correct box plots get wrong (regarding the outliers).
I was able to find one example which causes this behaviour - which is shown above.
